### PR TITLE
[NL] filter out pc charts for temperature and precipitation sv's

### PR DESCRIPTION
**Pull request by beets**
image

### CODE REVIEW

File: server/lib/nl_page_config.py [hunk 1]
```diff
@@ -169,8 +184,8 @@ def _topic_sv_blocks(category: subject_page_pb2.Category,
       sub_svs = extended_sv_map[sv]
       if not sub_svs:
         continue
-      sub_svs_exist = filter(lambda x: x in sv_exists_list, sub_svs)
-      if not sub_svs_exist:
+      sub_svs_exist = list(filter(lambda x: x in sv_exists_list, sub_svs))
+      if len(sub_svs_exist) == 0:
```
Comment (jehangiramjad) [inline server/lib/nl_page_config.py:188]: nit: can you do if sub_svs_list:

**Review by jehangiramjad — APPROVED** [review]

File: server/lib/nl_page_config.py (hunk 1 above)
Comment (beets) [reply server/lib/nl_page_config.py]: done

File: server/lib/nl_page_config.py [hunk 2]
```diff
@@ -341,37 +371,39 @@ def build_page_config(detection: Detection, data_spec: DataSpec,
...
-          # first check if "best" or "worst"
-          if RankingType.BEST in classifier.attributes.ranking_type:
-            tile.ranking_tile_spec.show_lowest = True
-          elif RankingType.WORST in classifier.attributes.ranking_type:
-            tile.ranking_tile_spec.show_highest = True
+      if _should_add_percapita(primary_sv):
+        tile = column.tiles.add()
+        sv_key = primary_sv + "_pc"
+        tile.stat_var_key.append(sv_key)
+        if classifier.type == ClassificationType.RANKING:
```
Comment (juliawu) [inline server/lib/nl_page_config.py:378]: I think moving `if classifier.type == ClassificationType.RANKING:` under `if _should_add_percapita(primary_sv):` means we won't ever show ranking charts for the weather data. Is this intended behavior?

Comment (beets) [reply server/lib/nl_page_config.py:378]: not for per capita. there's another block above for non-percapita

Comment (juliawu) [reply server/lib/nl_page_config.py:378]: gotcha. thanks for clarifying!

**Review by juliawu — APPROVED** [review]

**Pull request merged**